SilasKenneth/fast-food-fast-ui
# [Finishes #160135735] An admin user can edit an existing item

#### What this pull request does
Creates the page on the admin dashboard where an admin can edit an existing item
##### Description of task to be completed
 Create a page where an admin user can edit fast food items
    -  Add links items orders account and log out
    - Create the form to edit an item

##### Relevant pivotal tracker story
[#160135735](https://www.pivotaltracker.com/n/projects/2194200/stories/160135735)
##### How to test
  - Clone the repo ``git clone https://github.com/SilasKenneth/fast-food-fast-ui.git``
  -  Go to the fast-food-fast-ui directory and head to UI folder, ```git checkout the ft-admin-edit-item-160135735```
  - Open the edititem.html file in your browser and that should work

#### Screenshot
![screencapture-localhost-3000-edititem-html-2018-08-31-16_17_05](https://user-images.githubusercontent.com/11555354/44914897-27120180-ad3a-11e8-9c3a-b0946bbdfc04.png)
